sensors/01-meteorological-data.Rmd

```diff
@@ -68,31 +59,31 @@ The key is to process each type of met data (site, reanalysis, forecast, climate
 
 ### The Geostreams Database
 
-!(schema)[https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg]
+![Schema](https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg)
 
 #### Querying the API
 
-key query terms:
-
-* /sensors
-  * sensor_name=''
-* /streams
-  * 
-* /datapoints
-  * &since=YYYY-MM-DD
-  * &until=YYYY-MM-DD
-  * stream_id=
+The data can be accessed using a URL. You can find data for a particular station, 
+a certain sensor from a station, or the datapoints from a sensor. Below are
+the URLs for the UA-MAC AZMET weather station, a certain sensor at that station, 
+and datapoints from that sensor for the month of January 2017. 
 
-How to find the stream_id from a particular station or sensor:
+* Station: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors?sensor_name=UA-MAC+AZMET+Weather+Station
+* Sensor: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors/438/streams
+* Datapoints: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
 
-1. find streams on "UA-MAC AZMET Weather Station" https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors?sensor_name=UA-MAC AZMET Weather Station
-2. take sensor id from that response and find the streams: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors/438/streams
-3. take stream id from that and use in the datapoints query: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+All URLs have the same beginning (https://terraref.ncsa.illinois.edu/clowder/api/geostreams/), 
+then additional information is added for each type of data as shown below. 
 
+* Station: /sensors/sensor_name=[name]
```

Comment:
The example URL ends in a '/' so maybe get rid of that one above, or the leading ones here?

Reply:
See ae3aac94513e9b884a4113f6a3db8a6f8cae83d5